config.toml

```diff
@@ -44,5 +44,8 @@
   Folder = "players"
 
 [Resources]
+  # LoadPacks is true if packs should be loaded by the folder specified below. If this is false, then packs won't be
+  # attempted to be loaded from the folder, however you will still be able to add your own pack through the code.
+  LoadPacks = true
```

Comment:
Is this really useful? If you don't want to load packs, just don't add them to the folder?

Reply:
I can see it be useful in some cases, such as testing or simply just the fact that an empty resources folder isn't too elegant - especially if your server doesn't plan on using packs anyways. Up to you though.

Reply:
I think even with this setting in the majority of the cases the folder will be there anyway because people don't enable this from the start and honestly I think having an empty folder is fine. Feels a bit pointless to have a config setting just to prevent an empty folder.

Reply:
Removed.